-------------------------------------------------------------------------------
Test set: com.main.chatbot.ChatBotControllerTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 9.206 s <<< FAILURE! - in com.main.chatbot.ChatBotControllerTest
testGetBotReply(com.main.chatbot.ChatBotControllerTest)  Time elapsed: 8.791 s  <<< ERROR!
org.apache.http.conn.HttpHostConnectException: Connection to http://localhost:8085 refused
	at com.main.chatbot.ChatBotControllerTest.testGetBotReply(ChatBotControllerTest.java:13)
Caused by: java.net.ConnectException: Connection refused: connect
	at com.main.chatbot.ChatBotControllerTest.testGetBotReply(ChatBotControllerTest.java:13)

